gyrinx/core/static/core/scss/_classic_card.scss

```
@@ -673,6 +673,122 @@
+            gap: 0.6mm;
+            margin-bottom: 1.8mm;
+        }
+
+        .cc-detail .cc-label {
+            width: auto;
+        }
+
+        .cc-detail .cc-section__body {
+            // Prose, not a joined list — give the lines a little more air.
+            line-height: 1.25;
+            // Size to the text, and never grow — the shared rule grows the body
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)